Make it possible to run Zaplib initialize and callRust in Jest test (#33)
Zaplib/zaplib: zaplib/docs/src/basic_tooling.md

* Open Chrome DevTools, click the gear (⚙) icon in the top right corner of DevTools pane, go to the Experiments panel and tick **WebAssembly Debugging: Enable DWARF support**. (See also [this article](https://developer.chrome.com/blog/wasm-debugging-2020/)).

Note that these source maps read from hardcoded local file paths, so they'll only work on the computer that you've compiled on.

## Setup Javascript

### Jest

> Zaplib is possible to run using Jest testing framework. Following the [Integrating with JS](./tutorial_js_rust_bridge.md) tutorial let's assume we have a `sum` hook for `callRust` defined:

Reviewer: ```suggestion
Zaplib can run in the [Jest testing framework](https://jestjs.io/). Following [Tutorial: Integrating with JS](./tutorial_js_rust_bridge.md), let's assume we have a `sum` hook for `callRust` defined:
```

```rust,noplayground
// src/main.rs
use zaplib::*;

fn sum(values: &[u8]) -> u8 {
    values.iter().sum()
}

fn call_rust(name: String, params: Vec<ZapParam>) -> Vec<ZapParam> {
    if name == "sum" {
        let values = params[0].as_u8_slice();
        let response = vec![sum(&values)].into_param();
        return vec![response];
    }

    vec![]
}

register_call_rust!(call_rust);
```

> Create `jest.config.js` file with the following settings:

Reviewer: ```suggestion
Create a `jest.config.js` file with the following settings:
```

```js
/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
// eslint-disable-next-line no-undef
module.exports = {
  testEnvironment: "jest-environment-jsdom",
};
```

Then the simple Jest test would look like this:

```js
// example-jest.test.js

// Import polyfills for nodejs environment

> require("zaplib/dist/zaplib_nodejs_polyfill");

Reviewer: Maybe a few more sentences about this in the main text? E.g.:

Zaplib provides a set of polyfills for running in Node.js, which can be found in `zaplib/dist/zaplib_nodejs_polyfill` (in the `zaplib` npm package).
